OPHD/UI/StructureInspector.h

```diff
@@ -23,7 +23,10 @@ class StructureInspector : public Window
 private:
 	void btnCloseClicked();
 
+	void drawLabelAndValue(NAS2D::Point<int> position, const std::string& title, const std::string& text);
 	void drawPopulationRequirements();
+	void drawStructureTypeSpecific();
+	void drawResidenceText();
```

Comment:
Looks like these two methods have been removed.

Reply:
Thanks for catching.